stable/restore/values.yaml

```diff
@@ -53,21 +59,31 @@ admin:
 database:
   # name
   # NuoDB Database name.  must consist of lowercase alphanumeric
-  #characters '[a-z0-9]+'
+  # characters '[a-z0-9]+'
   name:
 
 restore:
-  # details of the database(s) involved in the backup
+  # name of the existing database being restored 
   target: ""
-#  source: ""
 
-  # backupSet
+  # source
   # Can be:
   # * the name of a backupset on a hotcopy-SM's backup disk;
   # * the metaname ':latest'
   # * a URL from which to download the backup (typically a TAR.GZ file)
-  backupSet: ""
+  source: ""
+
+  # credentials for the specified target - if different to the database.autoRestore.credentials
+  credentials: ""
 
+  # autoRestart
+  # if set to "true", running the restore chart causes the database to be restarted automatically
+  autoRestart: "true"
```

Comment:
Isn't it better the default to be `false`?

Reply:
@sivanov-nuodb 
Most users would expect that deploying the `restore` chart will *_cause_* a restore.
The `autoRestart = true` ensures that.

If the default were "false", then the default behaviour would be to set a flag that the next time the database is restarted, it should restore from backup - but it would not trigger that restart as part of deploying the `restore` chart.